xivo/tenant_helpers.py

```diff
@@ -148,6 +148,15 @@ def tenant_uuid(self) -> str | None:
     def user_uuid(self) -> str | None:
         return self._token_dict['metadata'].get('uuid')
 
+    def has_tenant_access(self, tenant_uuid: str | None) -> bool:
```

Comment:
Name suggestions:
* `tenant_allowed`
* `is_tenant_allowed`
* `has_tenant_permission`

No objections to the current name, just throwing alternatives if you prefer one.

Reply:
update to `is_tenant_allowed`